examples/machine-learning/a3-highgpu-8g/ml-slurm-a3-2-cluster.yaml

```diff
@@ -0,0 +1,257 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: slurm-a3-cluster
+
+terraform_backend_defaults:
+  type: gcs
+  configuration:
+    bucket: customer-tf-state-bucket  # modify to be a bucket owned and writable by customer
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: slurm-a3-cluster
+  region: customer-region
+  zone: customer-zone
+  server_ip_homefs: 0.0.0.0 ## MUST set to IP address of Filestore instance from base deployment!
+  remote_mount_homefs: /nfsshare
+  local_mount_homefs: /home
+  zones: []
```

Comment:
@mr0re1  will this need to be updated based on #2864 ?

Reply:
No, that's good.